code/modules/mob/living/carbon/xenomorph/abilities/queen/queen_powers.dm

```diff
@@ -1,6 +1,7 @@
 // devolve a xeno - lots of old, vaguely shitty code here
-/datum/action/xeno_action/onclick/deevolve/use_ability(atom/Atom)
+/datum/action/xeno_action/onclick/manage_hive/proc/de_evolve_other()
 	var/mob/living/carbon/xenomorph/queen/user_xeno = owner
+	plasma_cost = 500
```

Comment:
We're almost there, but if you do this it will set the plasma cost of the ability to 500 for the next time the queen uses the "manage hive" ability. You're better off setting plasma_cost to 0 on the ability and just making a new local var within these procs for the costs otherwise the cost for "manage hive" will be bouncing all over the place every time you use an ability.